package/kernel/button-hotplug/src/button-hotplug.c

```diff
@@ -85,6 +85,7 @@ static struct bh_map button_map[] = {
 	BH_MAP(BTN_9,		"BTN_9"),
 	BH_MAP(KEY_RESTART,	"reset"),
 	BH_MAP(KEY_POWER,	"power"),
+	BH_MAP(KEY_POWER2,	"reboot"),
```

Comment:
This change doesn't correspond to the commit description (base-files package). It should go into separate commit.